Podverse/PlayerHistoryItem.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  PlayerHistoryItem.swift
+//  Podverse
+//
+//  Created by Mitchell Downey on 5/21/17.
+//  Copyright © 2017 Podverse LLC. All rights reserved.
+//
+
+import UIKit
+
+class PlayerHistoryItem: NSObject, NSCoding {
+    let podcastfeedUrl: String!
```

Comment:
You don't really need the ! here. this is a readonly property and we are giving it a non-nil value in the class initializer so it is guaranteed to have some kind of value anyway. If you want to make it explicit that this specific class must always implement that specific variable, we can create a protocol but its not necessary here I believe. Just remove the ! and it should be just fine :)